common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -154,136 +154,176 @@
                 studentWordsKeys = [],
 
-            // Comma separated string of user enetered words.
+                // Comma separated string of user enetered words.
                 studentWordsStr,
 
-            // By default we do not scale.
+                // By default we do not scale.
                 scale = 1,
 
-            // Caсhing of DOM element
+                // Caсhing of DOM element
                 cloudSectionEl = this.wordCloudEl.find('.result_cloud_section'),
 
-            // Needed for caсhing of d3 group elements
+                // Needed for caсhing of d3 group elements
                 groupEl;
 
-        // If bounding rectangle is given, scale based on the bounding box of all the words.
+            // If bounding rectangle is given, scale based on the bounding box of all the words.
             if (bounds) {
                 scale = 0.5 * Math.min(
-                this.width / Math.abs(bounds[1].x - this.width / 2),
-                this.width / Math.abs(bounds[0].x - this.width / 2),
-                this.height / Math.abs(bounds[1].y - this.height / 2),
-                this.height / Math.abs(bounds[0].y - this.height / 2)
-            );
+                    this.width / Math.abs(bounds[1].x - this.width / 2),
+                    this.width / Math.abs(bounds[0].x - this.width / 2),
+                    this.height / Math.abs(bounds[1].y - this.height / 2),
+                    this.height / Math.abs(bounds[0].y - this.height / 2)
+                );
             }
 
             $.each(response.student_words, function(word, stat) {
                 var percent = (response.display_student_percents) ? ' ' + (Math.round(100 * (stat / response.total_count))) + '%' : '';
 
-                studentWordsKeys.push('<strong>' + word + '</strong>' + percent);
+                studentWordsKeys.push(HtmlUtils.interpolateHtml(
+                    '{listStart}{startTag}{word}{endTag}{percent}{listEnd}',
+                    {
+                        listStart: HtmlUtils.HTML('<li>'),
+                        startTag: HtmlUtils.HTML('<strong>'),
+                        word: word,
+                        endTag: HtmlUtils.HTML('</strong>'),
+                        percent: percent,
+                        listEnd: HtmlUtils.HTML('</li>')
+                    }
+                ).toString());
             });
-            studentWordsStr = '' + studentWordsKeys.join(', ');
+
+            studentWordsStr = '' + studentWordsKeys.join('');
 
             cloudSectionEl
-            .addClass('active')
-            .find('.your_words').html(studentWordsStr)
-            .end()
-            .find('.total_num_words').html(response.total_count);
+                .addClass('active');
+
+            HtmlUtils.setHtml(
+                cloudSectionEl.find('.your_words'),
+                HtmlUtils.HTML(studentWordsStr)
+            );
+
+            cloudSectionEl.find('.your_words').end().find('.total_num_words')
```

Comment:
The concatenation of response.total_count + " words submitted in total." is not good for translation. A placeholder for the number should be used.